macos/README.md

```diff
@@ -8,35 +8,35 @@ Wazuh
 
 Wazuh is an Open Source Host-based Intrusion Detection System that performs log analysis, file integrity monitoring, policy monitoring, rootkit detection, real-time alerting, active response, vulnerability detector, etc.
 
-In this repository, you can find the necessary tools to build a Wazuh package for OSX.
+In this repository, you can find the necessary tools to build a Wazuh package for macOS.
 
 ## Tools needed to build the package
 
 To build a Wazuh package you need to install the following tools:
-  - [Packages](http://s.sudre.free.fr/Software/Packages/about.html): You can install this on OSX using the generate_wazuh_packages.sh script in this repo.
-  - `git`: on OSX install with homebrew use `brew install git`
+- [Packages](http://s.sudre.free.fr/Software/Packages/about.html): You can install this on macOS using the generate_wazuh_packages.sh script in this repo.
+- `git`: on macOS install with homebrew use `brew install git`
 
-## Building OSX packages
+## Building macOS packages
 
-To build an OSX package, you need to download this repository and use the `generate_wazuh_packages.sh` script. This script will download the source code from the [wazuh/wazuh](https://github.com/wazuh/wazuh) repository and automatize the package generation.
+To build an macOS package, you need to download this repository and use the `generate_wazuh_packages.sh` script. This script will download the source code from the [wazuh/wazuh](https://github.com/wazuh/wazuh) repository and automatize the package generation.
```

Comment:
Change the last part from `...repository and automatize the package generation.` to `...repository and build the package`.